Update Payment Facilities

ialorro opened the pull request:
Deleted Transferwise from list of "Facilities not accepted at this time" @TQuock

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

ialorro committed:
Update Payment Facilities

Deleted Transferwise from list of "Facilities not accepted at this time" @TQuock

TQuock reviewed (changes requested):
Are we also going to add additional language as to who qualifies for Tranferswise?

justinegeffen committed:
Update how-to-get-paid.md

Editorial review for heading capitalization, formatting, and one punctuation change; changes committed.

justinegeffen reviewed (dismissed):
Looks good to me, thank you!

jasonblais commented:
/check-cla

jasonblais commented:
@ialorro heads up on the question from Tim above

ialorro commented:
Thank you @jasonblais ! This was resolved in pull request #136.

jasonblais commented:
Great! @TQuock if you can help approve if the changes look good, given #136

jasonblais committed:
Merge branch '0.2.1' into patch-2

justinegeffen commented:
Following up on this, @TQuock, are you happy for this to be merged? :)

TQuock commented on company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
 The person or legal entity the bank account is registered to must be in the name of the contracting individual/entity.
 
-_Example 1_: a staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith must have the following banking institution setup:
+_Example 1_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith must have the following banking institution setup:
```
```suggestion
_Example 1_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his surname, John Smith will need the following banking institution setup:
```

TQuock commented on company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
+* The bank account is his name or other known surname.
 
-_Example 2_: a staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:
+_Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:
```
```suggestion
_Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith will need the following banking institution setup:
```

TQuock commented on company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
 
 Primary Banking Institution and Bill.com
 
 The person or legal entity the bank account is registered to must be in the name of the contracting individual/entity.
```
```suggestion
The person or legal entity the bank account is registered to will need to be in the name of the contracting individual/entity.
```

TQuock replied:
@ialorro - can you please check my suggestions and make sure that you agree? If you are good, I'll commit these

TQuock commented on company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md:
```diff
@@ -18,23 +18,23 @@ If you are a vendor, refer to the [Bank Form](https://handbook.mattermost.com/co
+* The bank account is located in Spain.
+* The bank account is in the name of Better Coding, LLC.
 
 If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost requires banking information from a primary banking institution to be able to make reimbursements through Bill.com. Approved expense reports are reimbursed either by the 15th or end of each month. 
```
```suggestion
If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost needs banking information from a primary banking institution to be able to make reimbursements through Bill.com. Approved expense reports will follow our [Expense reimbursement policy](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#expense-reimbursement-policy).
```

ialorro commented:
duplicate pull request - resolved in #136

TQuock commented:
@ialorro did all the updates make it to the other commit?

ialorro commented:
@TQuock I believe so, this was the first pull request I created (not really knowing how to use GitHub) and then I created #136, which had all the edits needed for this section.

TQuock commented:
Thanks for the clarification. Issue is closed on my end

All the best,

*Tim Quock *
Director of Accounting
tim@mattermost.com | LinkedIn
<https://www.linkedin.com/in/timothy-quock-23300628/> | mattermost.com | google
calendar availability <https://calendly.com/matters/30min-1>

O: (650) 667-8512
C: (925) 878-1338



*"Train people well enough so they can leave, treat people well enough so
they don't want to" - Richard Branson *





On Wed, May 27, 2020 at 8:22 AM ialorro <notifications@github.com> wrote:

> @TQuock <https://github.com/TQuock> I believe so, this was the first pull
> request I created (not really knowing how to use GitHub) and then I created
> #136 <https://github.com/mattermost/mattermost-handbook/pull/136>, which
> had all the edits needed for this section.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mattermost/mattermost-handbook/pull/129#issuecomment-634735944>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMPAKUFIMPAJVXQSP2M47RTRTUVZ7ANCNFSM4MEF7VCA>
> .
>
